quupe-server/src/signup.ts

```diff
@@ -69,13 +70,15 @@ async function getUser(api: GraphQLClient, email: string): Promise<{ User }> {
 async function createGraphcoolUser(
     api: GraphQLClient,
     email: string,
-    password: string
+    password: string,
+    fullname: string
 ): Promise<string> {
     const mutation = `
-    mutation createGraphcoolUser($email: String!, $password: String!) {
+    mutation createGraphcoolUser($email: String!, $password: String!, $fullname: String!) {
       createUser(
         email: $email,
-        password: $password
+        password: $password,
+        fullname: $fullname
```

Comment:
I think in last pull request from Brandon, full name was removed. Just check with your group if you need this or not and make changes accordingly. Thank you.

Reply:
Yeah we do need the fullname in signup, I don't know why it was removed but we noticed that on the weekend and it was throwing errors when trying to sign up.

Reply:
ok good. no worries as long as it is working.